### DISCUSSION TITLE
how do I change the date format across the whole plugin

### DISCUSSION THREAD

**Discussion by sairajchouhan** [lewis6991/gitsigns.nvim#1131] [Q&A]
in line blame and blame, the date format is`YYYY-MM-DD` by default
how do I change it to `DD-MM-YYYY`?

**Comment by mtheunissen82** [comment]
For me more specifically about date formatting in file blame mode (not line blame).

The date format seems to fixed to YYYY-MM-DD . I tried to provide the `extra_opts` with value `--date=relative` as those extra_opts are supposed to be fed to `git blame` command (e.g. `:lua require("gitsigns").blame({extra_opts={"--date=relative"}}) `) but sadly that doesn't seem to do anything.

<img width="1143" height="245" alt="image" src="https://github.com/user-attachments/assets/fc8da5fd-3b9c-4491-b40f-d8fc79279119" />
